aio/tests/e2e/src/api-list.e2e-spec.ts

```diff
@@ -1,41 +1,58 @@
+import { by, element } from 'protractor';
 import { SitePage } from './app.po';
-import { element, by } from 'protractor';
 
 describe('api-list', () => {
   let page: SitePage;
+  const apiSearchInput = element(by.css('aio-api-list .form-search input'));
+  const apiStatusDropdown = element(by.css('aio-api-list aio-select[label="Status:"] .form-select-button'));
+  const apiTypeDropdown = element(by.css('aio-api-list aio-select[label="Type:"] .form-select-button'));
```

Comment:
Nit (moving uninitialized vars below initialized ones):

```suggestion
  const apiSearchInput = element(by.css('aio-api-list .form-search input'));
  const apiStatusDropdown = element(by.css('aio-api-list aio-select[label="Status:"] .form-select-button'));
  const apiTypeDropdown = element(by.css('aio-api-list aio-select[label="Type:"] .form-select-button'));
  let page: SitePage;
```